tests/test_WavefrontData.py

```diff
@@ -0,0 +1 @@
+from SimExLite.WavefrontData import WavefrontData 
```

Comment:
guess this needs to be completed but is  very important to test properly

Reply:
It will be addressed in this issue: https://github.com/PaNOSC-ViNYL/SimEx-Lite/issues/16